sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/tests/TestHelper.cs

```diff
@@ -210,7 +210,7 @@ public static AuthenticationEventsTriggerAttribute CreateAuthenticationEventTrig
         {
             return new AuthenticationEventsTriggerAttribute()
             {
-                TenantId = tenantId,
+                AuthorizedPartyAppId = tenantId,
```

Comment:
this cannot be correct. This has to be the first party app ID

Reply:
You are right, but this test doesn't check for those values. Updated the variable name and value to AuthorizedPartyAppId